pages/api/eval.ts

```diff
@@ -51,7 +51,26 @@ export async function rankEvalRequest(
     templates: [{ id, template: template.template }],
   }
 
-  return rankClient
+  if (name === 'negative') {
```

Comment:
Instead of setting this as the name, could we have it as part of the `ratings`?

e.g.
```JS
{languages: {
    invertMetric: true,
    examples: [
      { query: 'arbeiten', ratings: ['sr4kxmk3', 'utbtee43'] },
      { query: 'conosco', ratings: ['nnh3nh47'] },
      { query: 'allons', ratings: ['dqnapkdx'] },
    ],
    metric: {
      recall: {
        relevant_rating_threshold: 3,
        k: 100,
      },
    },
  }}
```

Reply:
Ideally, I'd like to contain the logic for the metric augmentation in the metric block itself, but I haven't come up with a neat way of doing that so far 😞